easybuild/easyconfigs/s/squashfs-tools/squashfs-tools-4.6.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'squashfs-tools'
+version = '4.6.1'
+
+homepage = ' https://github.com/plougher/squashfs-tools'
+description = """squashfs-fuse"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/plougher/squashfs-tools/archive']
+sources = ['master.tar.gz']
+
+build_cmd = "cd squashfs-tools ; make"
```

we can use the `start_dir` parameter instead of the `cd` cmd. we will also skip the configure step here and set the install prefix:
```suggestion
start_dir = "squashfs-tools"

skipsteps = ['configure']

installopts = 'INSTALL_PREFIX=%(installdir)s'
```